src/hostapi/wmme/pa_win_wmme.c

```diff
@@ -1832,8 +1831,6 @@ static PaError InitializeWaveHandles( PaWinMmeHostApiRepresentation *winMmeHostA
             ((HWAVEOUT*)handlesAndBuffers->waveHandles)[i] = 0;
     }
 
-    /* @todo at the moment we only use 16 bit sample format */
```

Comment:
This was added when supporting AC3 and SPDIF passthrough. We have to make sure we don't break that.

https://github.com/PortAudio/portaudio/commit/1de1b421f97302064ce2e30c7604e185a28da1ee

Reply:
Good point. I just checked and it doesn't look like this change can break this use case, because existing callers have to specify `paInt16` anyway, making this PR a no-op in this case.

https://github.com/PortAudio/portaudio/blob/68e963a990da19bb013133dcbad59c2ed8ea0cf9/examples/paex_wmme_ac3.c#L167

It makes sense that callers would have to use `paInt16`, because otherwise conversion would take place, resulting in corruption of the bitstream. So we can safely assume that every bitstreaming user is already passing `paInt16` here anyway, and will not be broken by this PR.